doc/content/microreactors/gcmr/index.md

```diff
@@ -6,4 +6,6 @@
 
 [Multiphysics Models](gcmr/GCMR_Multiphysics_models.md)
 
-[Results - Dynamic Multiphysics simulation of flow blockage](gcmr/GCMR_results.md)
\ No newline at end of file
+[Dynamic Multiphysics simulation of flow blockage](gcmr/GCMR_result_FlowBlockage.md)
+
```

Comment:
@aabdelhameedd - why is it `result_FlowBlockage` and `resultS_RIA`? (plural vs singular) Should we make it consistent?

Reply:
I have renamed `GCMR_result_FlowBlockage.md` to `GCMR_results_FlowBlockage`